spec/services/schedule_rules/open_hours_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ScheduleRules::OpenHours do
+  let(:product) do
+    create(
+      :setup_instrument,
+      :always_available
+    )
+  end
+  let(:schedule_rules) { product.schedule_rules }
+
+  it { expect(described_class.weekdays).to eq Date::ABBR_DAYNAMES }
+
+  before do
+    product.schedule_rules.destroy_all
+  end
+
+  context "#per_weekday groups open hours by weekday" do
+    let(:subject) { described_class.new(schedule_rules).per_weekday }
+
+    before do
+      create(
+        :schedule_rule,
+        :weekday,
+        product:,
+        start_hour: 9,
+        start_min: 0,
+        end_hour: 17,
+        end_min: 0,
```

Comment:
I think we don't need this as these are the default values

Reply:
I think it make sense to have these set if using a string literal as expectation value